src/components/Tags/Tag.utils.ts

```diff
@@ -1,42 +1,50 @@
 import { Tag } from "../../features/entity.types";
 
-export const defaultTagsList: string[] = [
+export const DefaultTagsList: string[] = [
   "pests",
   "disease",
   "sowed",
   "seedling",
+  "default",
 ];
 
-export const convertToTag = (tag: string) => {
-  switch (tag) {
+export const convertToTag = (tagName: string) => {
```

Comment:
dunno something more specific without having to look at the args to figure out what its converting 
```suggestion
export const tagNameToTag = (tagName: string) => {
```